configs/wdika/mridc-release.yaml

```diff
@@ -0,0 +1,25 @@
+target_repository:
+  HTTPS: https://github.com/wdika/mridc
+  # checkout a particular branch or a tag
+  checkout: main
+  # define installing package extras
+  install_extras: all
+  # copy some  test from the target repository
+  copy_tests:
+    - tests
+
+before_install:
+  - sudo apt-get install -y libsndfile1
+
+dependencies:
+  - name: pytorch-lightning
+    HTTPS: https://github.com/PyTorchLightning/pytorch-lightning.git
+    checkout: release/1.5.x
+    install_extras: loggers
+
+testing:
+  # additional pytest arguments
+  pytest_args: -m "not pleasefixme and not torch_tts" --cpu --with_downloads --relax_numba_compat
```

Comment:
is this really needed?

Reply:
Hi @Borda, thanks for the quick review. My backend is based on NVIDIA's NeMO and I do want to test numba compatibility as well. Do you think it's not needed?